src/references/debris_physics.md

```diff
@@ -0,0 +1,83 @@
+
+- Explosions
+
+    Having a group of items that scatter after an explosion, which works really well with **Damageable Objects**.
+
+Here's an example video of a damageable truck exploding and sending its parts everywhere:
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/DebrisPhysics/DebrisPhysics_ExplodingCar.mp4" type="video/mp4" />
+    </video>
+</div>
+
+## Where to find Debris Physics
+
+The **"Use Debris Physics"** option is found inside of a **Static Mesh's Properties** window under the **Physics** section.
+
+!!! note
+    The option will not be available unless the **Static Mesh** is a child component of a **Client Context**.
+
+!!! warning
+    Not all **Static Mesh** support **Debris Physics**. In those cases, the **Physics** section will not appear in the **Properties** window.
+
+![!Debris Physics](../img/DebrisPhysics/DebrisPhysics_Properties.png){: .center loading="lazy" }
+
+## Debris Physics Options
+
+The **Physics** section of the **Static Mesh's Properties** window has some settings to change the **Debris Physics**. Here they are explained below:
+
+| Property Name | Description | Notes |
+| --- | --- | --- |
+| Use Debris Physics | Toggle for activating the Debris Physics option. | Can only be activated in a Client Context. |
+| Override Mass | Toggle to manually change the mass of an object. | A default mass is calculated and used if inactive. |
+| Mass | The custom mass to override the default mass. | Mass measured in kilograms. |
+| Angular Damping | Drag force added for angular (rotational) movement. | Greater value results in a quicker stop, zero value results in no change, and negative value results in more movement over time. |
+| Linear Damping | Drag force added for linear (positional) movement. | Greater value results in a quicker stop, zero value results in no change, and negative value results in more movement over time. |
+
+## Adding Debris Physics
+
+1. Add a **Static Mesh** into the **Hierarchy** (example: `Ball - Soccer 01` from **Core Content**).
+2. Right click the **Static Mesh** and select **Create Network Context > New Client Context Containing This**.
+3. Select the **Static Mesh** again and open the **Properties** window.
+4. Activate the **Use Debris Physics** property under the **Physics** section.
+5. Enter **Preview Mode** and interact with the **Static Mesh**.
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/DebrisPhysics/DebrisPhysics_Soccer.mp4" type="video/mp4" />
+    </video>
+</div>
+
+## Learn More
+
+[StaticMesh](../api/staticmesh.md) | [PhysicsObject](../api/physicsobject.md) | [Vehicles](../ref/vehicles.md) | [DamageableObject](../api/damageableobject.md)
```

Comment:
How about a link to the networking reference down here for the user to explore more about contexts, or learn more about what a client context is that was mentioned in this ref?